Docs/container_diagram.mmd

```diff
@@ -0,0 +1,21 @@
+ C4Context
+      title Container diagram for LLM Benchmark System
+      Boundary(b0, "Dutch Government") {
+        System(web_api, "Web Application", "Frontend application for the users to interact with the system")
+        SystemDb(DB_LLM, "Database LLMs", "Stores all the LLMs")
```

Comment:
Do we need this? Or do we mean ephemeral cache of models?

Reply:
hmm yeah the LLMs would be ephemeral cache potentially (or maybe something that is able to store longer than just the lifetime of the pod, as we would download continously otherwise). But the LLM Benchmarks (if maintained by us) would probably still need a DB or the results of the the Benchmark runs also.

Reply:
Or we could also opt that even if we provide benchmarks we could store it on huggingface which would be a more standardized way and these could even be included in the llm open dashboard from huggingface itself

Reply:
Exactly, I would keep benchmarks separate (possibly, but not necessarily on huggingface)

Reply:
btw, this discussion seem to refer to the comment below. But indeed, the same should hold for both models and benchmarks in my opinion.

Reply:
Yes, we could use a LRU cache